src/realizations/catchment/Bmi_C_Formulation.cpp

```diff
@@ -21,10 +21,17 @@ std::string Bmi_C_Formulation::get_formulation_type() {
  */
 std::shared_ptr<Bmi_C_Adapter> Bmi_C_Formulation::construct_model(const geojson::PropertyMap& properties) {
     // First examine properties to see if other input variable values are provided
+    auto library_file_iter = properties.find(BMI_REALIZATION_CFG_PARAM_OPT__LIB_FILE);
```

Comment:
I wonder if it is reasonable to use standard library paths and library naming conventions to simplify this configuration.  Pointing to an exact library can be quite useful, so I wouldn't drop that, but say a BMI realization is defined in  the configuration that doesn't provide an explicit path, we could search `LD_LIBRARY_PATH` or explicit directories like `/lib` a  library file named `lib<bmi_name>.so`

Reply:
That's a good idea.  I think this should suffice for now, but I opened issue #221 to keep track of doing that in the future.